langchain/src/vectorstores/milvus.ts

```diff
@@ -236,7 +236,11 @@ export class Milvus extends VectorStore {
     const results: [Document, number][] = [];
     searchResp.results.forEach((result) => {
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      const fields = { pageContent: "", metadata: {} as Record<string, any> };
+      const fields = {
+        pageContent: "",
+        metadata: {} as Record<string, any>,
+        [this.primaryField]: 0,
```

Comment:
Why add this?

Reply:
@jacoblee93 I added it because we can delete rows from Milvus by primary field, and thus I added this field is returned from query. 

[https://milvus.io/docs/delete_data.md](https://milvus.io/docs/delete_data.md)

Reply:
Ah ok, I think the `0` was a bit confusing. Can we change it to `null` or `undefined` to be a bit more standard?

Reply:
Wait sorry I don't see how `fields[this.primaryKey]` is getting set?

Reply:
@jacoblee93  It generated automatically (we don't define it when we create a document), but I attached the primary key to metadata (after the document was created). 

I think maybe to create a some function that handling with regular query (without similarity) due to extract the primary key, something like that: 

[https://milvus.io/docs/query.md](https://milvus.io/docs/query.md)

What do you think?

Reply:
@jacoblee93 I fixed it.